assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss

```diff
@@ -103,6 +104,7 @@
     color: $color;
     opacity: 1; // Override Firefox's unusual default opacity; see https://github.com/twbs/bootstrap/pull/11526
   }
+  &:-ms-input-placeholder { color: $color; } // Internet Explorer 10+
```

Comment:
Let's forget about IE10+ in comments please.

-ms-input-placeholder this is something very 100% IE specific and is not missed or used by other browsers , I removed this previously, why would we bring this back?  My goal was to exclude IE and do only minimal changes where there was absolutely no doubt involved.